test/es-module/test-esm-detect-ambiguous.mjs

```diff
@@ -4,8 +4,8 @@ import { spawn } from 'node:child_process';
 import { describe, it } from 'node:test';
 import { strictEqual, match } from 'node:assert';
 
-describe('--experimental-detect-module', { concurrency: true }, () => {
-  describe('string input', { concurrency: true }, () => {
+describe('--experimental-detect-module', { concurrency: !process.env.TEST_PARALLEL }, () => {
+  describe('string input', { concurrency: !process.env.TEST_PARALLEL }, () => {
```

Comment:
Since `concurrency` is inherited from the parent, do we need to enable it twice here?

Reply:
I'll be honest, I only did a quick search and replace, I didn't look closely at the code. I'm tempted to not bother, even if it's redundant, it doesn't harm.